packages/common/src/types/field.ts

```diff
@@ -151,7 +159,8 @@ export const isTableCalculation = (
     item
         ? !('binType' in item) &&
           !!item.sql &&
-          !('type' in item) &&
+          !('description' in item) &&
+          !('tableLabel' in item) &&
```

Comment:
Table calculation icons in filters always displays the `numbers` icon
I believe it is because of this which is causing `isTableCalculation` to return `false` in `getFieldIcon` function `packages/frontend/src/components/common/Filters/FieldIcon.tsx`


https://github.com/lightdash/lightdash/assets/22939015/a2dbd301-a6aa-4835-a5e4-5640abe7cb1d

Reply:
yes, I can't get this combination right, we have a problem identifying types based on their fields...

Reply:
Fixed: 

![Screenshot from 2024-04-24 15-09-36](https://github.com/lightdash/lightdash/assets/1983672/1d3fa2a4-82c7-4932-85e8-c9f4c417fb45)